test/test_helper.rb

```diff
@@ -21,3 +21,11 @@
 require 'webmock/minitest'
 
 require 'mocha/minitest'
+
+# Override the cache/config dirs with the current repo's temp folder, as we don't want tests to touch the production
+# ones.
+ShopifyCli.send(:remove_const, :CACHE_DIR)
```

Comment:
Would it be feasible to have these configurable rather than redefining the constants?

Reply:
Does minitest indicate in any way that we're running tests? If we can tell that we can just change the actual definition of the consts, which would remove the need for patching.